### PR TITLE
[Bugfix:InstructorUI] Fixed Edit Gradeables page

### LINKED ISSUE #6859: Tabs on edit gradeable cover up footer
**Describe the bug**
On the create/edit gradeable page (especially when viewed on a small screen), there is extra space above the div with the content of the page, and if there is enough content displayed to fill the whole screen, that div hides the Submitty footer (either partially or fully, depends on the length of the page)
Screenshot_10

**Expected behavior**
The space above the page content div should be consistent with the rest of the pages on the site, and if the page is too long, the user should be able to scroll in order to see the footer.

### PR DESCRIPTION
**Pull request by immortalcodes**
### What is the current behavior?
Fixes #6859, Earlier on create/edit gradeable page the side navigation panel contained scroll and if the content on the page increased in the size then the footer went hidden under the content.

### What is the new behavior?
The scroll from the side navigation is removed and is made consistent with general user interface of the application.

### Other information?
Here is the screenshot for same
image

### CODE REVIEW

File: site/public/css/electronic.css [hunk 1]
```diff
@@ -27,7 +27,7 @@ main#main.full-screen-mode .content {
 
 #wrapper {
     padding-top: 30px;
-    min-height: 0;
```
Comment (shailpatels) [inline site/public/css/electronic.css:30]: Question: Is this min-height in this file needed ? The `global.css` will put the `#wrapper` CSS rule on every page with the `min-height: max-content;`. 

When I see it on firefox it seems the duplicate rules get ignored and the others are applied. Not sure if this is the same on other browsers. Would just removing the `min-height` in the electronic.css cause any issues instead?

**Review by Eceptonsu — COMMENTED** [review]
Works for me on Chrome, but I do agree that only having `the min-height: max-content;` in `global.css` will do the work.

File: site/public/css/electronic.css (hunk 1 above)
Comment (immortalcodes) [reply site/public/css/electronic.css:30]: @Eceptonsu @shailpatels  I also though that global.css must do the work but ..
image
Here in the screenshot (firefox browser) , It derives the css component from electronic.css and without it , the property didn't worked.
Hence, addition there was done.

**Review by Eceptonsu — CHANGES_REQUESTED** [review]
I think the original issue #6859 also mentioned that there is extra space between the header and the content div, maybe this should be fixed too.

**Comment by immortalcodes** [comment]
@Eceptonsu Thanks for reminding , I have removed that too.

**Review by Eceptonsu — APPROVED** [review]

**Pull request merged**